scripts/ci/credscan/CredScanSuppressions.json

```diff
@@ -144,7 +144,8 @@
         "src\\containerapp\\azext_containerapp\\tests\\latest\\cert.pfx",
         "src\\containerapp\\azext_containerapp\\tests\\latest\\test_containerapp_commands.py",
         "src\\containerapp\\azext_containerapp\\tests\\latest\\test_containerapp_env_commands.py",
-        "src\\containerapp\\azext_containerapp\\tests\\latest\\recordings\\test_containerapp_registry_msi.yaml"
```

Comment:
these don't seem to be new tests, but maybe re-recorded? Is there a reason we need to add these to CredSupressions with the new changes? as in have there been changes in API that is adding some secrets recently?

Reply:
We had two tests with the same name so we had to rename one and rerecord it. All the containerapp tests need a credscan suppression because of the log analytics keys that are used during the creation of containerapp environments